specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/networkWatcher.json

```diff
@@ -3480,19 +3480,16 @@
         "roundTripTimeMin": {
           "description": "Minimum roundtrip time in milliseconds.",
           "readOnly": true,
-          "format": "int32",
```

Comment:
why do we remove this format?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/9868/files - bugfix is needed